CHANGELOG.md

```diff
@@ -5,16 +5,20 @@ Notable changes to this project are documented in this file. The format is based
 ## [Unreleased]
 
 Breaking changes:
+- Added support for PureScript 0.14 and dropped support for all previous versions (#10, #14)
 
 New features:
+- Updated implementation to use Leibniz equality (#11)
```

Comment:
I _believe_ that #11 is backwards-compatible, but I could be mistaken. cc: @hdgarrood

Reply:
Yep 👍